resharper/resharper-unity/src/Application/UI/Help/ShowUnityHelp.cs

```diff
@@ -76,7 +85,11 @@ private FileSystemPath GetDocumentationRoot()
         {
             var appPath = mySolutionsManager.Solution?.GetComponent<UnityVersion>().GetActualAppPathForSolution();
             var contentsPath = UnityInstallationFinder.GetApplicationContentsPath(appPath);
-            return contentsPath.Combine(@"Documentation/en");
+            var root = contentsPath.Combine("Documentation");
+            var englishRoot = root.Combine("en");
+            if (!englishRoot.ExistsDirectory)
+                return root.GetChildDirectories().FirstOrDefault(englishRoot);
+            return englishRoot;
```

Comment:
Oh wow. Does this mean they're installing non-English documentation now?

Reply:
Not really. It can be downloaded separately and manually put to the folder
https://docs.unity3d.com/Manual/OfflineDocumentation.html